### PR TITLE
[top] Latest ast integration

### PR DESCRIPTION
**Pull request by tjaychen**
- hookup rng_fips to entropy_src
- properly name test_voltage/test_mode ports and hook them up
- hookup analog test connections to the pad

Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

File: hw/top_earlgrey/rtl/autogen/top_earlgrey.sv
Comment (mwbranstad) [inline hw/top_earlgrey/rtl/autogen/top_earlgrey.sv:2061]: At first I was wondering about the _o suffix but this signal appears to leave the "top core" module and connects to the AST module. This appears correct, but the limited context in this git review makes it a small challenge to follow. LGTM.

File: hw/top_earlgrey/rtl/top_earlgrey_asic.sv
Comment (arnonsha) [inline hw/top_earlgrey/rtl/top_earlgrey_asic.sv:598]: Looks OK

**Review by arnonsha — APPROVED** [review]
LGTM

File: hw/top_earlgrey/rtl/top_earlgrey_asic.sv
Comment (arnonsha) [inline hw/top_earlgrey/rtl/top_earlgrey_asic.sv:600]: Connectivity to AST ports is OK

**Comment by tjaychen** [comment]
it would be great if i can get one more LGTM from someone with merge rights.

**Review by eunchan — APPROVED** [review]

**Pull request merged**
